src/time_integration/paired_explicit_runge_kutta/methods_PERK2.jl

```diff
@@ -8,12 +8,6 @@ using LinearAlgebra: eigvals
 @muladd begin
```

Comment:
We should move the comments of the the `@muladd begin` macro to the macro, i.e., below the imports.